CMakeLists.txt

```diff
@@ -121,12 +121,29 @@ add_test(NAME CGMatrixTests COMMAND CGMatrixTests)
 add_test(NAME cholesky_dense COMMAND cholesky_test)
 add_test(NAME cholesky_crs COMMAND test_cholesky_CRS)
 
+# Define the source and destination paths for the Resources directory symlink
+set(RESOURCES_DIR "${CMAKE_SOURCE_DIR}/Resources")
+set(SYMLINK_DIR "${CMAKE_BINARY_DIR}/Resources")
+
+# Check if the symlink or directory already exists
+if(NOT EXISTS "${SYMLINK_DIR}")
+  # Create a symbolic link in the build folder to the Resources directory in the src folder
+  execute_process(
```

Comment:
... and that's a better way to create symbolic links with cmake compared to what I added in https://github.com/Leenkh3/Firefly/blob/66dcbc8eff4556f1fb7d07543fa8caa0624d4cb9/cmake/add_regression_test.cmake#L8 because your solution is portable, since cmake abstracts this away and will call the correct (potentially different) command on different operating systems.